lib/ansible/module_utils/common/file.py

```diff
@@ -49,57 +49,60 @@ class FileLock:
     '''
     Currently FileLock is implemented via fcntl.flock on a lock file, however this
     behaviour may change in the future. Avoid mixing lock types fcntl.flock,
-    fcntl.lockf and module_utils.common.file.FileLock as it will certainly cause
+    fcntl.flock and module_utils.common.file.FileLock as it will certainly cause
     unwanted and/or unexpected behaviour
     '''
     def __init__(self):
         self.lockfd = None
 
     @contextmanager
-    def lock_file(self, path, tmpdir, lock_timeout=None):
+    def lock_file(self, path, lock_timeout=None):
         '''
         Context for lock acquisition
         '''
         try:
-            self.set_lock(path, tmpdir, lock_timeout)
+            self.set_lock(path, lock_timeout)
             yield
         finally:
             self.unlock()
 
-    def set_lock(self, path, tmpdir, lock_timeout=None):
+    def set_lock(self, path, lock_timeout=None):
```

Comment:
Set same default timeout here:
```suggestion
    def set_lock(self, path, lock_timeout=15):
```